src/commands/containers/selectContainer.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { ContainerTreeItem } from '../../tree/containers/ContainerTreeItem';
+
+export async function selectContainer(context: IActionContext): Promise<string> {
+
+    let node: ContainerTreeItem;
+
+    //Expecting running containers to change often as this is a debugging scenario.
+    await ext.containersTree.refresh();
+
+    node = await ext.containersTree.showTreeItemPicker(/^(running)Container$/i, { ...context });
```

Comment:
Verify what happens on a cancel.  Pretty sure Eric would have set it up to throw usercancelled, but haven't inspected the code.

Reply:
Tried it, nothing exploded. Checked which code path was invoked, here's where the debugger lead me: https://github.com/microsoft/vscode/blob/dedf0de944fae56f0bf748913cbc1b2cc24987d2/src/vs/workbench/contrib/debug/browser/debugService.ts#L371-L374 
Returns false, all seem good!